lithops/standalone/backends/aws_ec2/aws_ec2.py

```diff
@@ -319,14 +318,14 @@ def is_ready(self, verbose=False):
         Checks if the VM instance is ready to receive ssh connections
         """
         login_type = 'password' if 'password' in self.ssh_credentials and \
-            not self.public else 'publickey'
+                                   not self.public else 'publickey'
         try:
             self.get_ssh_client().run_remote_command('id')
         except LithopsValidationError as e:
             raise e
         except Exception as e:
             if verbose:
-                logger.debug(f'SSH to {self.private_ip} failed ({login_type}): {e}')
+                logger.debug(f'SSH to {self.public_ip} failed ({login_type}): {e}')
```

Comment:
@aitorarjona I think this log is also shown within the master VM when it tries to communicate with the workers through the private IP. Maybe this should be `self.public_ip or self.private_ip` or something like this